lang_sam_tracker:
  ros__parameters:
    sam_model: sam2.1_hiera_small
    text_prompt: white line. road.
    box_threshold: 0.3
    text_threshold: 0.25